src/v2/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkActions.tsx

```diff
@@ -377,23 +386,50 @@ export const UtilButton: React.ForwardRefExoticComponent<
           <ActionIcon {...rest} fill="currentColor" />
         </Flex>
 
-        {label && (
-          <Text variant="xs" lineHeight={1}>
-            {label}
-          </Text>
-        )}
+        <UtilButtonInnerText
+          label={label}
+          longestLabel={longestLabel}
+          variant="xs"
+          lineHeight={1}
+        />
       </Component>
     )
   }
 )
 
+const UtilButtonInnerText: React.FC<UtilButtonInnerTextProps> = ({
+  label,
+  longestLabel,
+  ...rest
+}) => {
+  if (!label) {
+    return null
+  }
+
+  if (longestLabel) {
+    return (
+      <Box position="relative">
+        <VisibleText className="visibleText" {...rest}>
+          {label}
+        </VisibleText>
+        <HiddenText role="presentation" {...rest}>
```

Comment:
I _think_ you want `aria-hidden="true"` since you want to remove more than just the semantics?